plotly/offline/offline.py

```diff
@@ -137,7 +161,8 @@ def _plot_html(figure_or_data, show_link, link_text,
     return plotly_html_div, plotdivid, width, height
 
 
-def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
+def iplot(figure_or_data,
+          config={'showLink': True, 'linkText': 'Export to plot.ly'},
```

Comment:
we try really hard not to break backwards incompatibility. if we remove `show_link` and `link_text` then our user's code could break when they upgrade plotly. in this case, we might want to do something like:

```
iplot(figure_or_data, show_link=True, link_text='Export to plot.ly', **config_options)
```